src/testing/debugging.md

```diff
@@ -8,275 +8,26 @@ description: How to debug your app using the DevTools suite.
-as this is the quickest and least painful way of tracking
-down problems.
-
-For more information, see [Using the Dart analyzer][].
-
-## Logging
-
-Another useful debugging tool is logging.
-You set logging up [programmatically][logging]
-then view the output in the DevTools
-[logging view][], or in the console.
-
-## Debugging application layers
-
-Flutter was designed with a layered architecture that includes
-widget, rendering, and painting layers. For links to more
-information and videos, see [The Framework architecture][] on the
-[GitHub wiki][], and the community article, [The Layer Cake][].
-
-The Flutter widget inspector provides a visual representation
-of the widget tree, but if you want a greater level of detail,
-or you want a verbose text-based dump of the widget,
-layer, or render trees, see
-[Debug flags: application layers][]
-in the [Debugging Flutter apps programmatically][] page.
-
-## Debug mode assertions
-
-During development, you are highly encouraged to use Flutter's
-[debug mode][]. This is the default if you use bug icon in
-Android Studio, or `flutter run` at the command line.
-Some tools support assert statements through the
-command-line flag `--enable-asserts`.
-
-In this mode, Dart assert statements are enabled,
-and the Flutter framework evaluates the argument
-to each assert statement encountered during execution,
-throwing an exception if the result is false.
-This allows developers to enable or disable invariant
-checking, such that the associated performance cost
-is only paid during debugging sessions.
-
-When an invariant is violated, it's reported to the
-console, with some context information to help track
-down the source of the problem.
-
-For more information, check out [Assert][] in the
-[Dart language documentation][].
 
-## Debugging animations
 
-The easiest way to debug animations is to slow them down.
-The [Flutter inspector][] provides a **Slow Animations** button,
-or you can [slow the animations programmatically][].
-
-For more information on debugging janky (non-smooth)
-applications, see [Flutter performance profiling][].
-
-## Measuring app startup time
-
-To gather detailed information about the time it takes for your
-Flutter app to start, you can run the `flutter run` command
-with the `trace-startup` and `profile` options.
-
-```
-$ flutter run --trace-startup --profile
-```
-
-The trace output is saved as a JSON file called `start_up_info.json`
-under the `build` directory of your Flutter project.
-The output lists the elapsed time from app startup to these trace
-events (captured in microseconds):
-
-+ Time to enter the Flutter engine code.
-+ Time to render the first frame of the app.
-+ Time to initialize the Flutter framework.
-+ Time to complete the Flutter framework initialization.
-
-For example:
-
-```
-{
-  "engineEnterTimestampMicros": 96025565262,
-  "timeToFirstFrameMicros": 2171978,
-  "timeToFrameworkInitMicros": 514585,
-  "timeAfterFrameworkInitMicros": 1657393
-}
-```
```

Comment:
still relevant? @sfshaza2  is all this deleted documentation tracked somewhere else on the flutter website?

Reply:
Linked this PR to the issue: see previous comment